### ISSUE TITLE
New-SSHSession : Sequence contains more than one element

### ISSUE THREAD

**Issue by Adesh007** [darkoperator/Posh-SSH#106]
I am attempted to automate my login to my Cisco routers and the following error is throw. Appreciate you input, thanks. I am running powershell 3 on a window 7 Pro machine. 

VERBOSE: Using SSH Username and Password authentication for connection.
VERBOSE: Fingerprint for x.x.x.x: 
VERBOSE: Fingerprint matched trusted fingerprint for host x.x.x.x
New-SSHSession : Sequence contains more than one element
At C:\Users\xxxxxx\Desktop\Untitled2.ps1:20 char:1
- New-SSHSession x.x.x.x -AcceptKey -Credential $creds -Verbose
- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  - CategoryInfo          : InvalidOperation: (Renci.SshNet.SshClient:SshClient) [New-SSHSession], InvalidOperationException
  - FullyQualifiedErrorId : SSH.NewSshSession

**Comment by darkoperator** [comment]
What is the code inside the script?

**Comment by Adesh007** [comment]
Hello Sir, the following is my code. Just trying to validate that I can get into "enable mode" on the Cisco router:

Import-Module Posh-SSH
$filepath ="C:\Users\xxxxxxxx\Desktop\1\"
$username ="xxxxxxxx"
$password ="xxxxxxxx"
 
#Set Creds
$cred = new-object -typename System.Management.Automation.PSCredential `
         -argumentlist $username, $password
 
#Build SSH session
New-SSHSession X.X.X.X -AcceptKey -Credential $creds -Verbose
 
#Build open stream for use in cisco devices
$session = Get-SSHSession -Index 0
$stream = $session.Session.CreateShellStream("dumb", 0, 0, 0, 0, 1000)
$stream.Write("enable`n")
$stream.Write("enablepassword`n")
Remove-SSHSession -Index 0

**Comment by kjesso** [comment]
I'm getting the same error. Note I replaced the actual server name with "serverName".

**Input:**
`New-SSHSession -ComputerName "serverName" -Credential (Get-Credential)`

**Output:**
```
New-SSHSession : Sequence contains more than one element
At line:1 char:1
+ New-SSHSession -ComputerName "serverName" -Credential (Get-Credential ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (Renci.SshNet.SshClient:SshClient) [New-SSHSession], InvalidOperationException
    + FullyQualifiedErrorId : SSH.NewSshSession
```

**I'm running Windows 7 SP1 with PowerShell version 5:**

```
PSVersion: 5.0.10586.117
PSCompatibleVersions: {1.0, 2.0, 3.0, 4.0...}
BuildVersion: 10.0.10586.117
CLRVersion: 4.0.30319.42000
WSManStackVersion: 3.0
PSRemotingProtocolVersion: 2.3
SerializationVersion: 1.1.0.1
```

**And the Linux server "serverName" I'm trying to connect is running:**

`Red Hat Enterprise Linux Server release 5.11 (Tikanga)`

**Comment by darkoperator** [comment]
Are providing a FQDN or IP? Also did you tested with another client like putty?

**Comment by kjesso** [comment]
I tried connecting to a Macbook Pro via SSH and it connected successfully, I was just using the DNS name, not the FQDN. I then tried the original server I was trying to connect to but instead of the DNS name, I used the IP address and it connected successfully. I then removed the session and re-tried with the original DNS name and it again connected successfully. So I'm not sure what the issue was. Nothing was changed (ex: ip address, dns name, username or password).
